rastervision/data/vector_source/mbtiles_vector_source_config.py

```diff
@@ -62,11 +62,6 @@ def validate(self):
                 'MBTilesVectorSourceConfigBuilder requires uri which '
                 'can be set using "with_uri".')
 
-        if '{z}/{x}/{y}' not in self.config.get('uri'):
-            raise rv.ConfigError(
-                'MBTilesVectorSourceConfigBuilder requires uri containing '
-                '{z}/{x}/{y}.')
-
         if self.config.get('zoom') is None:
```

Comment:
Yes, it could be anything if it's not a ZXY schema. The `with_uri` method has a comment to this effect. If that's insufficient and you had something else in mind let me know.